content/02.introduction.md

```diff
@@ -1,41 +1,52 @@
 ## Introduction {.page_break_before}
 
-### General background about GitHub
+### General introduction to the use of collaborative computational resources by scientists
 
-*Contributors to this section: RCO, SSHS*
+*Contributors to this section: PHPB*
 
-With over 73 million registered users, GitHub and its underlying version control system Git, are a widely used platform for collaboration on computer code [@https://github.com/about].
-GitHub has become an indispensable tool for software developers because, through version control, users can track changes to multiple files and folders over time [@doi:10.1080/00031305.2017.1399928].
-Thus, users have an "audit trail" on the files they choose to store on their GitHub repositories which is simultaneously less *ad hoc* than passing files back and forth yet able to scale up as projects take on more files or collaborators [@doi:10.1186/1751-0473-8-7].
-GitHub as useful software and code repository service can broadly facilitate openly available source code alongside concomitant collaborative development.
-[@doi:10.1371/journal.pcbi.1004947]
+Most scientists, including ecologists and evolutionary biologists, have become critically dependent on computational tools in their research [see @doi:10.1109/SECSE.2009.5069155].
+Researchers now write and use code as part of their research workflow to perform a wide-variety of tasks facilitating scientific elaboration, ranging from data management, data analysis, study replication, to the application and the development of tools for hypothesis testing.
```

Comment:
Should this maybe include some mention of reproducibility as a motivation and/or benefit for using code?